specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json

```diff
@@ -122,13 +129,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/sensitivityLabels": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/columns/{columnName}/sensitivityLabels/recommended/enable": {
+      "post": {
         "tags": [
           "SensitivityLabels"
         ],
-        "description": "Gets the sensitivity labels of a given database",
-        "operationId": "SensitivityLabels_ListByDatabase",
+        "description": "Enables sensitivity recommendations on a given column",
+        "operationId": "SensitivityLabels_Enable",
```

Comment:
Shouldn't be SensitivityLabels**Recommendations**_Enable?

Reply:
Or "SensitivityLabels_EnableRecommendation" (like you did in MI)?

Reply:
Yes, will fix